### PR TITLE
Collapse display of large author lists

### PR DESCRIPTION
**Pull request by bdc34**
https://culibrary.atlassian.net/browse/ARXIVNG-924

### CODE REVIEW

File: browse/services/search/search_authors.py [hunk 1]
```diff
@@ -26,6 +26,31 @@ def is_divider(item: str)-> bool:
     return re.match(r'^(,|:)', item) is not None
 
 
+def split_long_author_list(authors: List[Union[str, Tuple[str, str]]], size:int) -> Tuple[List[Union[str, Tuple[str, str]]], List[Union[str, Tuple[str, str]]]]:
```
Comment (mhl10) [inline browse/services/search/search_authors.py]: 🏳️ I surrender!

File: browse/controllers/abs_page/__init__.py [hunk 2]
```diff
@@ -73,7 +74,8 @@ def get_abs_page(arxiv_id: str,
         abs_meta = metadata.get_abs(arxiv_id)
         response_data['abs_meta'] = abs_meta
         response_data['meta_tags'] = meta_tag_metadata(abs_meta)
-        response_data['author_links'] = queries_for_authors(abs_meta.authors)
+        response_data['author_links'] = \
+            split_long_author_list(queries_for_authors(abs_meta.authors), truncate_author_list_size)
```
Comment (mhl10) [inline browse/controllers/abs_page/__init__.py]: Unless I had this as an example, I would have trouble figuring out how to invoke this function. Can the function signature be simplified?

File: browse/services/search/search_authors.py (hunk 1 above)
Comment (bdc34) [reply browse/services/search/search_authors.py]: I've added a type alias.

**Comment by bdc34** [comment]
Merges ARXIVNG-924 to develop

**Pull request merged**